### ISSUE TITLE
Wrong conversion of timestamps to date

### ISSUE THREAD

**Issue by sheymann** [gephi/gephi#656]
Way to reproduce:

Open a new project, go to the Data Lab, Import Spreadsheet containing:
"source","target","date"
"cternes","jpwenzel","1331447778"

Import date column as either string or integer.
Then click on Merge Columns, select date, select Create Time Interval. Click OK on the final panel.

Date is 1970-01-16 instead of 2012-03-11, which is given by any timestamp converter elsewhere. The reason is because Gephi doesn't handle timestamps of 10 digits. It handles only timestamps of 13 digits.

**Comment by mbastian** [comment]
This is still happening in Gephi 0.9.1 cc @eduramiba

**Comment by eduramiba** [comment]
Right, we expect milliseconds for timestamps in all places (while inputting data in cells, imports, etc).
Not sure what we should do to support both.

**Comment by mbastian** [comment]
Ok then we can tag that issue as a wishlist to support 10 digits timestamps. We should document that thought

**Comment by duncdrum** [comment]
would padding the timestamp values work? Something similar to the invisible ````+Inf```` for `````@start | @end```` e.g.:
````format-number($timestamp, '0000000000000')````

**Comment by eduramiba** [comment]
Yeah, just adding 3 zeros will work.
